.github/workflows/minitest.yml

```
@@ -17,6 +17,14 @@ jobs:
   run-minitest:
     name: Run Minitest
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ci_node_index: [
```

Might be nice to rename this variable

```suggestion
        test_paths: [
```